lib/health_cards/exporter.rb

```diff
@@ -14,14 +14,39 @@ def download(jws)
       # Export JWS for $issue-health-card endpoint
       # @param [Array<JWS>, String] An array of JWS objects to be exported
       # @return [String] JSON string containing a FHIR Parameters resource
-      def issue(jws)
-        params = jws.map { |j| FHIR::Parameters::Parameter.new(name: 'verifiableCredential', valueString: j) }
+      def issue(fhir_params)
+        *jws = yield extract_types!(fhir_params)
+
+        params = jws.compact.map { |j| FHIR::Parameters::Parameter.new(name: 'verifiableCredential', valueString: j) }
+
         FHIR::Parameters.new(parameter: params).to_json
       end
 
       def qr_codes(jws)
         QRCodes.from_jws(jws)
       end
+
+      private
+
+      def extract_types!(fhir_params)
+        if fhir_params.nil?
+          code = 'structure'
+          err_msg = 'No Parameters found'
+        elsif !fhir_params.valid?
```

Comment:
I don't think the validation in fhir models is robust enough that we should rely on it in the library.

Reply:
I looked this up and the only validation it does is check that each FHIR::Parameter has a populated name attribute. I could code that manually or remove this line if you prefer but I don't think it's going to cause any issues as is.